src/VisualStudio/IntegrationTest/IntegrationTests/VisualBasic/BasicKeywordHighlighting.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Shared.TestHooks;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Microsoft.VisualStudio.IntegrationTest.Utilities;
+using Xunit;
+
+namespace Roslyn.VisualStudio.IntegrationTests.Basic
+{
+    [Collection(nameof(SharedIntegrationHostFixture))]
+    public class BasicKeywordHighlighting : AbstractEditorTest
+    {
+        protected override string LanguageName => LanguageNames.VisualBasic;
+
+        public BasicKeywordHighlighting(VisualStudioInstanceFactory instanceFactory)
+            : base(instanceFactory, nameof(BasicKeywordHighlighting))
+        {
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.Classification)]
+        public void NavigationBetweenKeywords()
+        {
+            VisualStudio.Editor.SetText(@"
+Class C
+    Sub Main()
+        For a = 0 To 1 Step 1
+            For b = 0 To 2
+        Next b, a
+    End Sub
+End Class");
+
+            Verify("To", 3);
+            VisualStudio.ExecuteCommand("Edit.NextHighlightedReference");
+            Assert.Equal(VisualStudio.Editor.GetCaretPosition(), 112);
```

Comment:
Can you use SetUpEditor and then get the position from the $$ in that markup? I really dislike these large numbers that represent positions in code.